.github/workflows/test.yml

```diff
@@ -143,7 +143,7 @@ env:
   CONTACT_US_EMAIL_ADDRESS: ${{ vars.CONTACT_US_EMAIL_ADDRESS }}
   FF_TFM_PAYMENT_RECONCILIATION_ENABLED: "true"
   FF_TFM_FACILITY_END_DATE_ENABLED: "true"
-  GEF_DEAL_VERSION: ${{ vars.GEF_DEAL_VERSION }}
+  GEF_DEAL_VERSION: 1
```

Comment:
I'm not familiar with the versioning - what's the advantage of having this hard coded / could it be using `vars.GEF_DEAL_VERSION`?

Seems that in the future, if there is a new version this could be missed